index.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>learning gits</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+
+  <body>
+    <main id="main">
```

Comment:
Great use of semantic tags 👍 👍 , I think you could have used more HTML5 semantic tags as obtainable in best Html5 practice.

Reply:
Good job 👍 identifying the use of the HTML5 semantic tag. However, as of this project requirement, there is a lot of `div` tags used in this project they can be replaced by `section` tags. Kindly mention that up and also you can comment about the excessive use of `id` they can be replaced by `class` since we use `id` to refer to unique parts in our code base.